src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -99,7 +99,7 @@ export default function SnapshotListModal() {
     data = { data: [], meta: { count: 0, limit: 20, offset: 0 } },
   } = useGetSnapshotList(uuid as string, page, perPage, debouncedSearchQuery, sortString);
 
-  const { data: contentData } = useFetchContent([uuid]);
+  const { data: contentData } = useFetchContent([snapshotUUID]);
```

Comment:
Same as above, no need to change this value here, uuid in this case is actually repoUUID, we want to make sure we fetch the correct repo, a snapshot uuid here wouldn't work.